frank-flow/src/frontend/src/app/shared/services/flow-structure.service.ts

```diff
@@ -346,20 +346,18 @@ export class FlowStructureService {
     if (this.attributeListIsEmpty(pipeline.attributes)) {
       this.createFirstAttribute(firstPipeAttribute, pipeline, true);
     } else {
-      this.editSingleAttribute(firstPipeAttribute, pipeline.attributes, true);
+      this.editSingleAttribute(firstPipeAttribute, pipeline.attributes);
     }
   }
 
   editSingleAttribute(
     changedAttribute: ChangedAttribute,
-    attributeList: FlowNodeAttributes,
-    flowUpdate = false
+    attributeList: FlowNodeAttributes
   ): void {
     const editOperations: monaco.editor.IIdentifiedSingleEditOperation[] = [];
     const editAttributeOperation = this.editAttribute(
       changedAttribute,
-      attributeList,
-      flowUpdate
```

Comment:
Denk wel dat dit nog nodig is toch? Wat deden die andere functies die editAttribute aanroepen?